### PR TITLE
DaveTheHeadcrab - Fluff items

### PR DESCRIPTION
**Pull request by DaveTheHeadcrab**
Removes Makkota Atani's fluff item. This was coded but never actually added to the database.

Adds generic single-use fluff tattoo gun, can apply tattoos to any human mob that's not an IPC.

Removes the check for species on the update_markings proc, this served no function other than sanity and restricted the ability to add markings in-game (As any marking allowed by your species in the datum is automatically available to you in the setup menu)

Adds Elliot Campbell's fluff items:
 - Nylon windbreaker
 - Ear piercing
 - Circuitry tattoo gun

:cl: DaveTheHeadcrab
tweak: Removes the check for species on the update_markings proc.
/:cl:

### CODE REVIEW

**Comment by johannhawk** [comment]
this reminds me of my tattoo sprites i made a long time ago...

**Comment by KasparoVy** [comment]
Glad to see my markings bit getting some more use, even if it is just for fluff.

**Comment by DaveTheHeadcrab** [comment]
@KasparoVy 
example

In-game it will be blue, but here it is.

**Comment by KasparoVy** [comment]
Looks pretty wicked

**Comment by DaveTheHeadcrab** [comment]
UPDATE:
Removes a check restricting body markings to species that are able to select them in the setup menu in update_markings.dm, this does not effect who can select body markings in the setup menu, only if you can have your icon updated to *have* said markings at any point throughout the round.

**Comment by DaveTheHeadcrab** [comment]
Also DNM as fuck, tat gun is being tested now.

**Comment by KasparoVy** [comment]
Might be a good idea to try and put in some sort of delay for the tattoo gun, with a welder sound since it's relatively close (unless we actually have a tattoo sound)

**Comment by DaveTheHeadcrab** [comment]
There is a delay, and the welder sound is in but not committed yet.

**Comment by KasparoVy** [comment]
Oh, wicked

**Comment by DaveTheHeadcrab** [comment]
@TheDZD 
Tested, good to merge.
Updating OP with changes now.

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -367,4 +416,34 @@
 	name = "combat pants"
 	desc = "Medium style tactical pants, for the fashion aware combat units out there."
 	icon_state = "chaps"
-	item_color = "combat_pants"
\ No newline at end of file
+	item_color = "combat_pants"
+
+/obj/item/clothing/suit/jacket/fluff/windbreaker // Anxipal: Kido Qasteth
```
Comment (KasparoVy) [inline code/modules/customitems/item_defines.dm]: Duplicate definition?

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -367,4 +416,34 @@
...
+	item_state = "elliot_windbreaker"
+	adjust_flavour = "zip"
+
+/obj/item/clothing/suit/jacket/fluff/windbreaker // DaveTheHeadcrab: Elliot Campbell
+	name = "nylon windbreaker"
+	desc = "A cheap nylon windbreaker, according to the tag it was manufactured in New Chiba, Earth.<br>The color reminds you of a television tuned to a dead channel."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "elliot_windbreaker"
+	item_state = "elliot_windbreaker"
+	adjust_flavour = "unzip"
```
Comment (KasparoVy) [inline code/modules/customitems/item_defines.dm]: If this isn't working already (totally forget this if it is), going to need to add ignore_suitadjust = 0 in order to have it actually be adjustable

**Comment by DaveTheHeadcrab** [comment]
I do. Hence the custom flavor for adjustment.

**Comment by KasparoVy** [comment]
Forgot that it's a child of jacket, lel

Nevermind!

**Comment by DaveTheHeadcrab** [comment]
@TheDZD @Fox-McCloud 
Boop

**Pull request merged**